packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -264,6 +264,30 @@ void main() {
       expect(picker.initialDateTime, isNotNull);
     });
 
+    testWidgets('initial date honors minuteInterval', (WidgetTester tester) async {
+        DateTime newDateTime;
+        await tester.pumpWidget(
+          CupertinoApp(
+            home: SizedBox(
+              width: 400,
+              height: 400,
+              child: CupertinoDatePicker(
+                onDateTimeChanged: (DateTime d) => newDateTime = d,
+                initialDateTime: DateTime(2018, 10, 10, 10, 3),
+                minuteInterval: 3,
+              )
+            )
+          )
+        );
+
+        // 03 -> 06
+        await tester.drag(find.text('03'), _kRowOffset);
+        await tester.pump();
+        await tester.pump(const Duration(milliseconds: 500));
```

Comment:
I'm surprised this is needed. If we dragged exactly by a row offset, we shouldn't need it to settle via an animation right? 

In general, if a wait duration is needed, the exact amount needed and reason should be documented.

Reply:
You're right it's not needed. I copied those lines from somewhere else in the file.